src/pages/docs/creators/ray/conversion-to-ray-on-golem-tutorial.md

```diff
@@ -0,0 +1,301 @@
+[WITHOUT RAY] deal count: 50 time: 0:00:07.907285
+```
+
+## Parallelize with Ray
+
+Let's now proceed to parallelization.
+
+We need to install ray :)
+```bash
+# Install ray
+pip install ray[default]
```

Comment:
if someone do this and then try to install `ray-on-golem` then might have issues with versions incompatibility

Reply:
hm... when I do `pip install ray`, and then separately `pip install ray-on-golem` in downgrades ray...
not sure if this is right?